corehq/motech/repeaters/models.py

```diff
@@ -1264,9 +1265,6 @@ def succeeded(self):
         # incorrect status code interpretation resulting in Empty state.
         return self.state == State.Success or self.state == State.Empty
 
-    def is_queued(self):
-        return self.state == State.Pending or self.state == State.Fail
-
```

Comment:
I kinda liked this helper 😅 any objection if I add it back in in a PR that refactors the repeat record report page? Obviously would reference the new constants instead of having this exact logic.